.wordlist.txt

```diff
@@ -14,6 +14,8 @@ Furo
 Git
 GitHub
 Grafana
+hotplug
```

Comment:
Can we move these words to .custom_wordlist.txt? This file is reserved for common words needed across projects (helps with easy maintenance).

Reply:
Done.